dev.py

```diff
@@ -261,10 +261,16 @@ def do_build(
     print("Creating ZIP archive.")
     shutil.make_archive("out/blenderkit", "zip", "out", "blenderkit")
 
+    # Handle multiple install locations
     if install_at is not None:
-        print(f"Copying to {install_at}/blenderkit")
-        shutil.rmtree(f"{install_at}/blenderkit", ignore_errors=True)
-        shutil.copytree("out/blenderkit", f"{install_at}/blenderkit")
+        # Convert single path to list for consistent handling
+        install_locations = install_at if isinstance(install_at, list) else [install_at]
```

Comment:
Not needed at all, spams the codebase. Even for single item the results is list (https://docs.python.org/3/library/argparse.html).

Reply:
Removed